README.md

```diff
@@ -95,3 +95,13 @@ end
 alias NewRelic.Instrumented.HTTPoison
 HTTPoison.get("http://www.example.com")
 ```
+
+#### Ignore transactions
```

Comment:
Would you mind removing the changes to the `README` file please? I'd like to keep the contents very high level

Reply:
Yeah, sure. The section was removed.